bin/rucio

```diff
@@ -1,24 +1,34 @@
 #!/usr/bin/env python
-
-"""
- Copyright European Organization for Nuclear Research (CERN)
-
- Licensed under the Apache License, Version 2.0 (the "License");
- You may not use this file except in compliance with the License.
- You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
-
- Authors:
- - Mario Lassnig, <mario.lassnig@cern.ch>, 2012-2018
- - Vincent Garonne, <vincent.garonne@cern.ch>, 2012-2018
- - Thomas Beermann, <thomas.beermann@cern.ch>, 2012, 2017
- - Yun-Pin Sun, <yun-pin.sun@cern.ch>, 2013
- - Cedric Serfon <cedric.serfon@cern.ch>, 2013-2018
- - Martin Barisits <martin.barisits@cern.ch>, 2013-2018
- - Joaquin Bogado <joaquin.bogado@cern.ch>, 2014, 2015
- - Evangelia Liotiri <evangelia.liotiri@cern.ch>, 2015
- - Tobias Wegner <tobias.wegner@cern.ch>, 2017
- - Nicolo Magini <nicolo.magini@cern.ch>, 2018
-"""
+# Copyright 2012-2018 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Mario Lassnig, <mario@lassnig.net>, 2012-2018
```

Comment:
Mario.Lassnig@cern.ch

Reply:
several mails in the git log history ?

Reply:
yep, few came from my private git account